core-web/apps/mcp-server/src/utils/response.ts

```diff
@@ -41,7 +43,11 @@ export function formatResponse(
         return message;
     }
 
-    const opts = { includeRawData: false, maxDataLength: 1000, ...options };
+    // Check for environment variable to control truncation
```

Comment:
The `formatResponse` function have a parameter for `options`, we're not using that, we can remove it.